webapp/packages/plugin-session-expiration/src/SessionExpireWarningDialog/SessionExpireWarningDialogBootstrap.ts

```diff
@@ -74,7 +72,7 @@ export class SessionExpireWarningDialogBootstrap extends Bootstrap {
         const { sessionState } = await this.graphQLService.sdk.sessionState();
 
         if (sessionState.valid) {
-          await this.sessionResource.refreshSilent();
+          this.sessionResource.touchSession();
```

Comment:
in case this async function will be finished with exception we will get 'uncaught exception error in console log'
please add try/catch or .then.catch